src/Common/perf/BenchmarksRunner/BenchmarksRunner.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <ProjectGuid>{013324FD-A4CE-4E56-B127-29474EA26FC}</ProjectGuid>
+    <OutputType>Exe</OutputType>
+    <!-- To be consistent between netfx and others we override the SDK behavior which defaults to .dll -->
+    <TargetExt>.exe</TargetExt>
+    <RootNamespace>BenchmarksRunner</RootNamespace>
+    <AssemblyName>BenchmarksRunner</AssemblyName>
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    <CopyNuGetImplementations>false</CopyNuGetImplementations>
+    <NoWarn>0436</NoWarn>
+    <IsTestProject>true</IsTestProject>
+    <IsBenchmarkDotNetProject>true</IsBenchmarkDotNetProject>
+    <Configurations>netstandard-Debug;netstandard-Release;</Configurations>
+    <ExcludeFromCodeCoverage>true</ExcludeFromCodeCoverage>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="BenchmarksRunner.cs" />
+    <Compile Include="CoreFxToolchain.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' != 'net46'">
```

Comment:
you can remove that condition as we never compiler against net46. The current netfx tfm is net472